jobs-builder/jobs/include/os2node.yaml.inc

```diff
@@ -0,0 +1,21 @@
+{#
+# Copyright (c) 2020 Red Hat, Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Convert OS name to node label string.
+#}
+{% if os == "centos8" -%}
+centos8_azure
+{% elif os == "debian10" -%}
+debian10_azure
+{% elif os == "fedora32" -%}
+fedora32_azure
```

Comment:
Update to F34/35?

Reply:
Unfortunately we are still running the Fedora job on Fedora 32, but Snir has worked to add Fedora 34.

Reply:
@wainersm, as Snir added a newer Fedora version, could you update this one?

Reply:
I will update properly, @fidencio . But notice that we are still triggering the Fedora 32 jobs on pull requests. I will talk with Snir so we finish that migration.

Reply:
done